packages/client/src/ee/containers/modelDetail.tsx

```diff
@@ -0,0 +1,152 @@
+        title: 'Models',
+        link: '/models?page=1'
+      },
+      {
+        key: 'model',
+        matcher: /\/models/,
+        title: `Model: ${modelName}`,
+        onClick: () => {getModel.refetch()}
+      }
+    ];
+
+    let pageBody;
+    if (getModel.error) {
+      pageBody = 'Cannot load model';
+    } else if (!model || !modelVersions) {
+      pageBody = <Skeleton />;
+    } else {
+      pageBody = this.renderModel(modelName, modelVersions, model, mlflow, getModel);
+    }
+
+    return (
+      <>
+        <PageTitle
+          breadcrumb={<Breadcrumbs pathList={breadcrumbs} />}
+          title={"Model Management"}
+        />
+        <PageBody>{pageBody}</PageBody>
+      </>
+    );
+  }
+
+  private renderModel(modelName: any, modelVersions: any, model: any, mlflow: any, getModel: { error?: any; loading: boolean; variables: { where?: any; }; refetch: Function; mlflow?: any; model?: any; modelVersions?: any; }) {
+    const columns: any = [{
+      title: 'Version',
+      dataIndex: 'version',
+      render: this.renderVersion(modelName),
+    }, {
+      title: 'Registered At',
+      dataIndex: 'creationTimestamp',
+      render: formatTimestamp,
+      sorter: (a, b) => compareTimestamp(a.creationTimestamp, b.creationTimestamp),
+      defaultSortOrder: 'descend',
+    }, {
+      title: 'Deployed By',
+      render: () => '',
+    }, {
+      title: 'Action',
+      render: () => <Button>Deploy</Button>,
+    }];
+    const data = modelVersions;
+
+    let pageBody = <>
+      <Row gutter={36}>
+        <Col span={20}>
+          <Field labelCol={4} valueCol={8} label='Created Time' value={formatTimestamp(model.creationTimestamp)} />
+          <Field labelCol={4} valueCol={8} label='Last Modified' value={formatTimestamp(model.lastUpdatedTimestamp)} />
+          <Field labelCol={4} valueCol={8} label='Description' value={formatTimestamp(model.description)} />
+        </Col>
+        <Col span={4}>
+          <div style={{ textAlign: 'right' }}>
+            <Button onClick={() => {
+              openMLflowUI(mlflow, `/#/models/${encodeURIComponent(modelName)}`);
+            } }>
+                         MLflow UI
+            </Button>
+          </div>
+        </Col>
+      </Row>
+      <Table
+        style={{ paddingTop: 8 }}
+        dataSource={data}
+        columns={columns}
+        rowKey="name"
+        loading={getModel.loading} />
+    </>;
+    return pageBody;
+  }
+}
+
+export default compose(
+  withRouter,
+  withGroupContext,
+  graphql(QueryModel, {
+    options: (props: Props) => {
+      const {groupContext, match} = props;
+      let {modelName} = match.params as any;
+      modelName = decodeURIComponent(modelName)
+      const group = groupContext ? groupContext.name : undefined;
+
+      return {
+        variables: {
+          group,
+          name: modelName
+        },
+        fetchPolicy: 'cache-and-network'
+      }
+    },
+    name: 'getModel'
```

Comment:
Tips: add option `alias: 'withGetModel'` will easier determine the graphql context in react inspector.